From jrm8005@gmail.com  Mon Nov 17 14:55:13 2008
From: "J.R. Mauro" <jrm8005@gmail.com>
Date: Thu, 13 Nov 2008 19:11:22 -0500
Subject: Staging: et131x: fix build failure
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <20081114001122.GA18839@tuxbook-pro>


Fix build failure in et131x driver - missing \ in multiline macro

Signed-off by: J.R. Mauro <jrm8005@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/et131x/et131x_debug.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/et131x/et131x_debug.h
+++ b/drivers/staging/et131x/et131x_debug.h
@@ -206,10 +206,10 @@
 #define DBG_ASSERT(C)						\
 	do {							\
 		if (!(C)) {					\
-			DBG_PRINT("ASSERT(%s) -- %s#%d (%s) ", \
+			DBG_PRINT("ASSERT(%s) -- %s#%d (%s) ",  \
 			     #C, __FILE__, __LINE__, __func__); \
 			DBG_TRAP;				\
-		}
+		}						\
 	} while (0)
 
 #define STATIC
